sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1474,13 +1483,9 @@ export class ApplicationService extends RecordDataModelService<Application> {
       },
       where: {
         id: applicationId,
-        applicationStatus: In([
-          ApplicationStatus.Assessment,
-          ApplicationStatus.Enrolment,
-          ApplicationStatus.Completed,
-        ]),
+        applicationStatus: In(applicationStatuses),
         student: {
-          id: studentId,
+          id: options?.studentId,
```

Comment:
what happens if the option is null

Reply:
`null` or `undefined`
![image](https://user-images.githubusercontent.com/61259237/224448897-c82cadef-6581-487e-a064-5370a814a8a4.png)

Student id provided
![image](https://user-images.githubusercontent.com/61259237/224448920-7f07eddd-5ff7-4084-a176-d79d02827b51.png)